game/scripts/vscripts/GameMode.ts

```diff
@@ -216,7 +215,7 @@ export class GameMode {
                 }
 
                 // Remove starting TP from player
-                Timers.CreateTimer(1/30, () => {
+                Timers.CreateTimer(1 / 30, () => {
```

Comment:
Can we just change this to either 0 or FrameTime()?

Reply:
that's not a change from this PR